src/libasr/codegen/asr_to_cpp.cpp

```diff
@@ -210,8 +210,11 @@ class ASRToCPPVisitor : public BaseCCPPVisitor<ASRToCPPVisitor>
     }
 
     std::string convert_variable_decl(const ASR::Variable_t &v, bool use_static=true,
-                                      bool use_templates_for_arrays=false)
+                                      bool use_templates_for_arrays=false, bool _=false)
     {
+        if( _ ) {
+
+        }
```

Comment:
What is this for?

I recommend not using default arguments, and just use `bool /* arg_name */`.